liam2/tests/output/demography_2002.csv

```diff
@@ -0,0 +1,19 @@
+agegroup,gender,,

```

Comment:
Good to avoid that error in the future, but this particular file should still be removed from the PR.